cookbook/creating-strongly-typed-json-collections/README.md

```diff
@@ -0,0 +1,75 @@
+# Creating Strongly Typed JSON Collections
+
+Sometimes, it might be useful to declare JSON and a corresponding code-behind that makes it possible to programmatically create collections that, when serialized to JSON, become documents that are collections of other JSON objects.
+
+In this example, it's described how to create a strongly typed collection of `Person` objects as created here:
+
+<div class="code-name">Person.json</div>
+
+```json
+{
+  "Name": ""
+}
+```
+
+<div class="code-name">Person.json.cs</div>
+
+```cs
+using Starcounter;
+
+namespace TypedJSONCollectionSample 
+{
+    partial class Person : Json 
+    {
+    }
+}
+```
+
+To start creating this collections, add a new `Starcounter Typed JSON with code-behind` template. In this example, it'll be named "PersonCollection".
```

Comment:
this => "such" or "these" and "add a new ... template" => "add a new item using the ... template".